test/entry_set_test.rb

```diff
@@ -66,9 +66,17 @@ def test_entries
   end
 
   def test_find_entry
-    # by default, #find_entry is case-sensitive
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('name1'))
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('NaMe1', false))
```

Comment:
@rastarobot's change was merged into master ONE DAY AGO. Fairly certain no "other gems" picked up his variant just yet, especially given there was no release since.